src/reduxStore/slices/toastsSlice.ts

```diff
@@ -1,13 +1,13 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { TRANSACTION_STATUS_TOAST_ID } from 'constants/transactionStatus';
 import { ToastsEnum } from 'types';
-import { logoutAction } from '../commonActions';
 import {
   CustomToastType,
   FailTransactionToastType,
   TransactionToastType
-} from 'types/toasts';
+} from 'types/toasts.types';
 import { getUnixTimestamp } from 'utils/dateTime/getUnixTimestamp';
-import { TRANSACTION_STATUS_TOAST_ID } from 'constants/transaction-status';
+import { logoutAction } from '../commonActions';
```

Comment:
LOGOUT_ACTION

Reply:
this is not a constant, it's an object created with @reduxjs/toolkit